scanpipe/templates/scanpipe/includes/project_list_table.html

```diff
@@ -16,6 +17,9 @@
       <th class="break-all">
         <a href="{{ project.get_absolute_url }}">{{ project }}</a>
       </th>
+      <td>
+        {{ project.created_date| date:"d-m-Y" }}
```

Comment:
What about using the natural style as we do on the details view?
https://github.com/nexB/scancode.io/blob/main/scanpipe/templates/scanpipe/project_detail.html#L18

Also, what about displaying this info directly under the project name, with the following style `has-text-grey is-size-7` instead of a column.

<img width="636" alt="Screenshot 2022-06-13 at 10 29 41" src="https://user-images.githubusercontent.com/489057/173292953-e00f37d5-5e87-4081-95bf-1c0036aadbf7.png">

Reply:
like this?

```
<th class="break-all">
    <a href="{{ project.get_absolute_url }}">{{ project }}</a><br>
    <span class="has-text-grey is-size-7" title="{{ project.created_date|date:'N j, Y, P T' }}">Created {{ project.created_date|naturaltime }}</span>
</th>
```

Reply:
Sure, but let's use the `is-block` class on the span instead of using `<br>` tag for layout.

Reply:
there is a text overflow for 97+ characters in the
```
/project/uuid/
/project/uuid/resources/
/project/uuid/resources/number/
```

what about using `is-widescreen` over `is-max-desktop`